### PR TITLE
Update with DTS hackathon info

### PR DESCRIPTION
**Pull request by mromanello**

### CODE REVIEW

File: _events/2021-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+The first DTS hackathon is organised by the DTS community and generously funded by the [dhCenter UNIL-EPFL](https://dhcenter-unil-epfl.com/) and the [École Nationale des Chartes](http://www.chartes.psl.eu/). The **goal** of this hackathon is to raise awareness about the [DTS specifications](https://w3id.org/dts), encourage their use and, ultimately, to improve the ecosystem of datasets and tools around DTS.
+
+## Format
+
+The hackathon will take place fully online between **27 September - 8 October 2021**.
+
+We propose two types of hacks:
+1. **data hacks**, aimed at exposing new or already existing textual resources via a DTS-compliant API with the goal of enlarging the variety of domains, languages and periods covered by DTS. Participants proposing a data hack *may* be affiliated with the institution owning the data, but this is not strictly necessary (e.g. in case of open data).
+2. **tool hacks**, aimed at enlarging the offer of DTS-compatible tools by extending existing software or developing new ones. Ideas for tool hacks include:
+- development of DTS client libraries in any programming language (e.g., Go, Java, Ruby);
```
Comment (PonteIneptique) [inline _events/2021-hackathon.md]: Should we add Javascript and Python, that are quite broadly used ? I feel like it should be there :)

File: _events/2021-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- development of generic viewers for DTS texts collections;
+- extending existing annotation tools (e.g. [Recogito](https://recogito.pelagios.org/), [INCEpTION](https://inception-project.github.io/)) to support DTS as a way of importing TEI documents to annotate;
+- adding integration with tools/libraries that perform linguistic and/or semantic enrichment of textual data such as [NLTK](http://nltk.org/) , [CLTK](http://cltk.org/) or [DHTK](https://dhtk.unil.ch/);
+- integration with digital editing tools such as collation tools (e.g., [CollateX](https://collatex.net/)) or publication tools (e.g., [EFES](https://github.com/EpiDoc/EFES), [ediarum](https://www.ediarum.org/))
```
Comment (PonteIneptique) [inline _events/2021-hackathon.md]: eXistDB / TEI Publisher ?

**Review by PonteIneptique — CHANGES_REQUESTED** [review]
I feel like the ownership of the products should be clearly stated: anything produced by the authors is the property of their author. We just require an Open License (including Non Commercial if needs be) ?
Two other small feedback added as well

**Comment by mromanello** [comment]
Thanks for all points, @PonteIneptique (also ping @PietroLiuzzo)

> I feel like the ownership of the products should be clearly stated: anything produced by the authors is the property of their author. We just require an Open License (including Non Commercial if needs be) ?

and especially this. We hadn't yet talked about it, but that's clearly something we need to be clear about upfront.

**Comment by PietroLiuzzo** [comment]
I like the idea that products should be openly licensed.

**Review by PietroLiuzzo — APPROVED** [review]

**Pull request merged**